CONTRIBUTING.md

```diff
@@ -154,6 +154,7 @@ Make sure that [Travis](http://www.travis-ci.org) greenlights the pull request w
  - use lower case with underscores for method names
  - it is generally preferred to use ASCII operators and identifiers over
    Unicode equivalents whenever possible
+ - function signature should not be followed by an empty line
```

Comment:
It would be good if the word `end` is highlighted, in order to make it clearer.

Even this looks good enough too, however highlighting it would make it clearer IMO.